plugins/certs/subcommands/print

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+set -eo pipefail
+[[ $DOKKU_TRACE ]] && set -x
+source "$PLUGIN_CORE_AVAILABLE_PATH/common/functions"
+source "$PLUGIN_AVAILABLE_PATH/certs/functions"
+
+cmd-certs-download() {
+  declare desc="exports SSL cert or key via STDOUT"
+  declare cmd="$1"
```

Comment:
I think this should be `certs:print`, right?

Reply:
It is now 'show SSL cert....' Is that what you meant?

Reply:
Well the cmd="$1" should be "certs:show".

Also I guess the description is out of date.

Reply:
The cmd="$1" was taken verbatim from: https://github.com/dokku/dokku/blob/master/plugins/certs/subcommands/add#L33

Reply:
fixed

Reply:
That one is weird due to symlinks, this one is better: https://github.com/dokku/dokku/blob/master/plugins/certs/subcommands/remove#L8